pyproject.toml

```diff
@@ -1,3 +1,47 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "geo-deep-learning"
+version = "0.1.0a0"
+description = "Geospatial deep learning framework for segmentation tasks"
+readme = "README.md"
+authors = [
+  { name = "Victor Alhassan", email = "victor.alhassan@NRCan-RNCan.gc.ca" },
+  { name = "Luca Romanini", email = "luca.romanini@NRCan-RNCan.gc.ca" },
```

Comment:
need to list all developers from our team who worked on that unless it's a fully new product